blockbuster/Cargo.toml

```diff
@@ -11,7 +11,7 @@ readme = "../README.md"
 [dependencies]
 spl-account-compression = { version = "0.2.0", features = ["no-entrypoint"] }
 spl-noop = { version = "0.1.3", features = ["no-entrypoint"] }
-mpl-bubblegum = { version = "0.10.1", features = ["no-entrypoint"] }
+mpl-bubblegum = { git = "https://github.com/metaplex-foundation/mpl-bubblegum.git", rev = "c03745be2dbfe142a60abfbb3d98ccf226d2e834" }
```

Comment:
Should we release a patch version of the client and reference the crate directly here?

Reply:
Yeah we could do that.  I have one more change added to https://github.com/metaplex-foundation/mpl-bubblegum/pull/53 that was needed to integrate with blockbuster.  Looking through Read API I don't see any other reason the Rust client would need to change to finish that integration (it just uses ID and some of the types).  But it may be good to at least have it working initially before publishing the updated client?

Reply:
Updated cargo.toml.

Thank you for publishing the 1.0.1-beta.1 Rust client.